internal/storage/testdata/test-board

```diff
@@ -0,0 +1,80 @@
+{
+    "tiles": [
+        [
+            {
+                "tile": {
```

Comment:
2 questions. 
- Since it's a json object, why the file don't have `.json` extension ? 
- Wouldn't it be easier to have just an array of object 
```
[
    {
        "shape": 1,
        "treasure": "A",
        "rotation": 180 
    }
]
```
?

Reply:
I did not add the `.json` to avoid extra formatting, but it can be added easily.
I used the same format for the JSON and the model to have an easier time reading the JSON file.